deps/BUILD.bazel

```diff
@@ -13,6 +13,7 @@ filegroup(
         "@xz",
         "@xz//:lzma",
         "@zlib//:libz_so",
+        "@libsepol//:sepol",
```

Comment:
This target won't work on macos. I've introduced a select in my PR to make smoketest work on all platforms. See example:
https://github.com/DataDog/datadog-agent/pull/41984/files